src/test/java/com/bravo/payment/service/PaymentServiceTest.java

```diff
@@ -0,0 +1,108 @@
+package com.bravo.payment.service;
+
+import com.bravo.App;
+import com.bravo.common.dao.model.mapper.ResourceMapper;
+import com.bravo.common.utility.PageUtil;
+import com.bravo.payment.dao.model.Payment;
+import com.bravo.payment.dao.repository.PaymentRepository;
+import com.bravo.payment.model.dto.PaymentReadDto;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static java.util.UUID.randomUUID;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@ContextConfiguration(classes = {App.class})
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+public class PaymentServiceTest {
+
+  @Autowired
+  private HttpServletResponse httpResponse;
+
+  @Autowired
+  private PaymentService paymentService;
+
+  @MockBean
+  private ResourceMapper resourceMapper;
+
+  @MockBean
+  private PaymentRepository paymentRepository;
+
+  final String userId = randomUUID().toString();
+  private List<PaymentReadDto> dtoPayments;
+
+  @BeforeEach
+  public void beforeEach() {
+    final List<Integer> ids = IntStream
+        .range(1, 10)
+        .boxed()
+        .collect(Collectors.toList());
```

Comment:
Is this final list still mutable?

Reply:
Yep. This was more habit, but also avoids the more likely case of reassigning the list by mistake.